x-pack/elastic-agent/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -602,7 +602,7 @@ func (c *enrollCmd) startAgent(ctx context.Context) (<-chan *os.ProcessState, er
 
 func (c *enrollCmd) stopAgent() {
 	if c.agentProc != nil {
-		c.agentProc.StopWait()
+		c.agentProc.StopWait() //nolint:errcheck // no error check here
```

Comment:
Could we check the error and log it at least instead of adding the nolint directive ?